chart/watermarkpodautoscaler/values.yaml

```diff
@@ -29,6 +29,11 @@ labelsAsTags: []
 podSecurityContext: {}
 # fsGroup: 2000
 
+ddProfiling:
+  enabled: false
+
+workers: 1
```

Comment:
IMO to be backward compatible the `workers` value should be empty. because here you will always add the new args that doesn't exist on previous version.

OR, you add an `if` in the template file. Something like

```yaml
            {{- if no eq .Values.workers 1 }}
             - -workers={{ .Values.workers }}
            {{- end }}     
```